routes/root.js

```diff
@@ -43,6 +43,9 @@ route.get('/leaderboard/:year?', async (req, res) => {
     let { year } = req.params
```

Comment:
`let { year = '2020' } = req.params`

just do this here.

Reply:
don't forget to run and test the solution.

Reply:
done..